frontend/package.json

```diff
@@ -3,85 +3,37 @@
   "version": "0.0.1",
   "private": true,
   "dependencies": {
-    "@babel/core": "7.6.0",
-    "@emotion/react": "^11.9.3",
-    "@emotion/styled": "^11.9.3",
-    "@material-ui/core": "^4.6.1",
-    "@material-ui/icons": "^4.5.1",
-    "@mui/icons-material": "^5.8.4",
-    "@mui/joy": "^5.0.0-alpha.39",
-    "@mui/material": "^5.9.2",
-    "@svgr/webpack": "4.3.2",
-    "@typescript-eslint/eslint-plugin": "^2.2.0",
-    "@typescript-eslint/parser": "^2.2.0",
-    "babel-eslint": "10.0.3",
-    "babel-jest": "^24.9.0",
-    "babel-loader": "8.0.6",
-    "babel-plugin-named-asset-import": "^0.3.4",
-    "babel-preset-react-app": "^9.0.2",
-    "camelcase": "^5.2.0",
-    "case-sensitive-paths-webpack-plugin": "2.2.0",
-    "classnames": "^2.2.6",
-    "css-loader": "2.1.1",
-    "customize-cra": "^0.8.0",
-    "dotenv": "6.2.0",
```

Comment:
Check required peer deps, please.
Seems like `dotenv` and a couple of others are required.
At least I could start the app in docker after getting these warns.

Reply:
fixed by means of updating config-overwrite script